modular_skyrat/code/modules/surgery/organs/tongue.dm

```diff
@@ -1,4 +1,10 @@
 /obj/item/organ/tongue/lizard/
 	modifies_speech = FALSE
 
-/obj/item/organ/tongue/lizard/handle_speech(datum/source, list/speech_args)
\ No newline at end of file
+((
+/obj/item/organ/tongue/lizard/handle_speech(datum/source, list/speech_args)
+/datum/modular_skyrat/language/datum/modular_skyrat/language/calcic
+		/datum/modular_skyrat/language/moffic
+		/datum/modular_skyrat/language/shadowtongue
+		/datum/modular_skyrat/language/sylvan
```

Comment:
/datum/modular_skyrat/language/moffic,
/datum/modular_skyrat/language/shadowtongue,
/datum/modular_skyrat/language/sylvan,

Reply:
Each language datum is separated by a , at the end